packages/flutter/test/material/slider_test.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('Slider can move when tapped', () {
+    testWidgets((WidgetTester tester) {
+      Key sliderKey = new UniqueKey();
+      double value = 0.0;
+
+      tester.pumpWidget(
+        new StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return new Material(
+              child: new Center(
+                child: new Slider(
+                  key: sliderKey,
+                  value: value,
+                  onChanged: (double newValue) {
+                    setState(() {
+                      value = newValue;
+                    });
+                  }
+                )
+              )
+            );
+          }
+        )
+      );
+
+      expect(value, equals(0.0));
+      tester.tap(tester.findElementByKey(sliderKey));
+      expect(value, equals(0.5));
+      tester.pump(); // No animation should start.
```

Comment:
This is a little confusing. Don't we expect the slider to animate from 0.0 to 0.5?